### ISSUE TITLE
Horizontal={false} not working in Android

### ISSUE THREAD

**Issue by tiennguyen9988** [leecade/react-native-swiper#738]
How to resolve error: Horizontal={false} not working in Android ? Thanks you!

**Comment by A1ik** [comment]
+1

**Comment by smitthakkar1** [comment]
Installl this: 
https://www.npmjs.com/package/@nart/react-native-swiper

This will solve the issue.

**Comment by Kylin1973** [comment]
Install 'npm i @nart/react-native-swiper' ?
This command seems doesn't work, Has somebody solved this issue?

**Comment by ravisojitra** [comment]
Yes, i solved it. This solves the verticle issue in react native.
1) npm i @nart/react-native-swiper --save
2) import Swiper from '@nart/react-native-swiper'
And this will work.
